tests/test_authentication.py

```diff
@@ -7,6 +7,7 @@
 class UserTestCases(unittest.TestCase):
     """
     Test successful registration
+    Test user registration twice
```

Comment:
Create a Base test so that you don't have to call setup method in each of your test files or test cases

Reply:
Okay, let me work on that.